core/types/pow/cuckoo.go

```diff
@@ -59,29 +58,19 @@ func ConvertBytesToUint32Array(data []byte) []uint32 {
     return nonces
 }
 
-// set scale ,the diff scale of circle
-func (this *Cuckoo) SetScale (scale uint32) {
-
-}
-
-//get scale ,the diff scale of circle
-func (this *Cuckoo) GetScale () int64 {
-    return 1856
-}
-
 //get cuckoo block hash bitarray with 42 circle nonces
 //then blake2b
-func (this *Cuckoo)GetBlockHash (data []byte) hash.Hash {
+func (this *Cuckoo)GetBlockData (data []byte) []byte {
     circlNonces := [cuckoo.ProofSize]uint64{}
     nonces := this.GetCircleNonces()
     for i:=0;i<len(nonces);i++{
         circlNonces[i] = uint64(nonces[i])
     }
-    return this.CuckooHash(circlNonces[:],int(this.GetEdgeBits()))
+    return this.CuckooHashData(circlNonces[:],int(this.GetEdgeBits()))
 }
 
 //calc cuckoo hash
-func (this *Cuckoo)CuckooHash(nonces []uint64,nonce_bits int) hash.Hash {
```

Comment:
if nobody uses this method, I think it can be combined with `GetBlockData()` or rename the method, any way you don't return a hash in this way. it's inconsistent